core/src/test/scala/com/stripe/dagon/DataFlowTest.scala

```diff
@@ -16,7 +18,7 @@ object DataFlowTest {
     def optionMap[U](fn: T => Option[U]): Flow[U] =
       Flow.OptionMapped(this, fn)
 
-    def concatMap[U](fn: T => TraversableOnce[U]): Flow[U] =
+    def concatMap[U](fn: T => IterableOnce[U]): Flow[U] =
```

Comment:
The tests don’t need to be binary compatible so you could change this to Iterable and simplify the compat stuff?

Up to you.

Reply:
👍